resources/monitoring/profile-production.yaml

```diff
@@ -35,4 +51,30 @@ kube-state-metrics:
       memory: "128Mi"
     limits:
       cpu: "500m"
-      memory: "1Gi"
\ No newline at end of file
+      memory: "2Gi"
+
+grafana:
+  persistence:
+    enabled: "false"
+    inMemory:
+      enabled: "false"
+  env:
+    GF_ALERTING_ENABLED: "false"
+    GF_AUTH_ANONYMOUS_ORG_ROLE: Viewer
+    GF_DASHBOARDS_MIN_REFRESH_INTERVAL: 5m
+  dataSources:
+    jaeger:
+      enabled: "false"
+  extraExposePorts: |-
+    - name: proxy-status
+      port: 15020
+      protocol: TCP
+      targetPort: 15020
+  sidecar:
+    dashboards:
+      searchNamespace: "kyma-system"
+    datasources:
+      searchNamespace: "kyma-system"
+  kyma:
+    authProxy:
+      configDocsLink: "https://help.sap.com/docs/BTP/65de2977205c403bbc107264b8eccf4b/3e4299cfd0884c428e6b4774225638e8.html"
```

Comment:
That whole block should not get in as it is specific to the SAP setup and will break compatibility from OS perspective